src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import duke.commands.*;
```

Comment:
It might be better to import the classes individually instead of doing a wildcard import :)

Reply:
Thanks for pointing that out! After configuring gradle, it seems that my intellij settings have resetted 😭